libs/spacegrep/src/test/Matcher.ml

```diff
@@ -202,25 +202,17 @@ let matcher_corpus =
       Matches [ "1 2"; "2 3" ],
       "$A ... $B",
       "1 2 3" );
-    (* See also "overlapping matches with open end" *)
     ( "overlapping matches with closed end",
       Matches [ "a b" ],
       "a ... b",
       "a a b" );
+    ("overlapping matches with open end", Matches [ "a a b" ], "a ...", "a a b");
     ("leading dots", Matches [ "a b" ], "... b", "a b c");
```

Comment:
Given "a ... b" and "a b a b" I would also expect that it either matches all of it ("a b a b") or maybe better that it produces two matches, both "a b".

Reply:
It would produce two matches, both "a b".

Reply:
```
$ ./bin/spacegrep 'a ... b' <(echo 'a b a b')
/dev/fd/63:a b a b

/dev/fd/63:a b a b
```